Dockerfile

```diff
@@ -1,3 +1,5 @@
+ARG ASYNCAPI_CLI_VERSION=0.50.0
```

Comment:
ok, now I see where `0.50` got from in the generator PR 😄 

anyway, why not latest?

Reply:
Copy pastes be like that. So let me copy the same answer 😆 

It's possible that the latest version will cache and you will end up with a docker image with an invalid generator version. With an argument, you can test past versions locally and ensure that you install the right generator version. Seems like a declarative approach.

Now that I think about it we should after the build actually "test" if the installed version inside the image is the same as release